src/ugeneui/src/project_support/ProjectImpl.cpp

```diff
@@ -119,6 +136,7 @@ void ProjectImpl::addDocument(Document* d) {
 
     connect(d, SIGNAL(si_objectAdded(GObject*)), SLOT(sl_onObjectAdded(GObject*)));
     connect(d, SIGNAL(si_objectRemoved(GObject*)), SLOT(sl_onObjectRemoved(GObject*)));
+    connect(d, SIGNAL(si_removeRelations(GObject*)), SLOT(sl_removeAllrelationsWithObject(GObject*)));
```

Comment:
Do not use SIGNAL/SLOT, use function pointers in new code.
Function pointers are validated by the compiler. SIGNAL/SLOT only in runtime